django/db/migrations/operations/models.py

```diff
@@ -12,6 +12,16 @@
 )
 
 
+def _check_for_duplicates(arg_name, objs):
+    used_vals = set()
+    for val in objs:
+        if val in used_vals:
+            raise ValueError(
+                'Found duplicate %s in CreateModel operation.' % arg_name
```

Comment:
I didn't follow the discussion closely but was there a reason for not including the duplicate value in the message? I thought that was in the first version of the message.